doc/conf.py

```diff
@@ -42,6 +42,11 @@
     print "ipython: %s, %s" % (IPython.__version__, IPython.__file__)
 except ImportError:
     print "no ipython"
+try:
+    import seaborn
```

Comment:
I also had an issue where building the docs on this branch popped up matplotlib windows which paused the build on my laptop. Not quite sure what's going on -- some import may be switching the matplotlib backend.

On Thu, Aug 6, 2015 at 11:24 AM, Joe Hamman <notifications@github.com>
wrote:

>> @@ -42,6 +42,11 @@
>>      print "ipython: %s, %s" % (IPython.__version__, IPython.__file__)
>>  except ImportError:
>>      print "no ipython"
>> +try:
>> +    import seaborn
> That is really odd. However, I just built the docs and was not able to reproduce your described behavior.
> What happens if you put `seaborn.reset_orig()`  after the import.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/xray/xray/pull/509/files#r36447826